recipe/meta.yaml

```diff
@@ -49,8 +52,16 @@ test:
     - huggingface_hub
   commands:
     - pip check
+    - cd bindings/python
+    # tests require multiprocessing with fork, which is not available on Windows.
+    - pytest tests --ignore="tests/documentation" -k "not test_continuing_prefix_trainer_mistmatch"  # [not win]
```

Comment:
Are there any tests that can be ran for Windows?

Reply:
add `not multiprocessing` for win?